internal/exec/resolvable/resolvable.go

```diff
@@ -207,40 +216,67 @@ func (b *execBuilder) makeObjectExec(typeName string, fields schema.FieldList, p
 	methodHasReceiver := resolverType.Kind() != reflect.Interface
 
 	Fields := make(map[string]*Field)
+	rt := unwrapPtr(resolverType)
 	for _, f := range fields {
-		methodIndex := findMethod(resolverType, f.Name)
-		if methodIndex == -1 {
+		methodIndex := -1
+		fieldIndex := -1
+
+		// 1) Use resolver type's method when
+		//	1.1) __Type and __Schema requests
+		// 	1.2) Or field has arguments
+		//	1.3) Or it is configured to use method
+		//	1.4) Or it is an interface
+		// 2) Otherwise use resolver type's field
```

Comment:
Please, reformat the comment as:
```
		// 1. Use method resolver when:
		//	a. __Type and __Schema requests; or
		// 	b. The requested field has arguments; or
		//	c. The schema is not configured to use field resolvers; or
		//	d. It is an interface
		// 2. Otherwise, use a field resolver
```